badge_server/test_badges.py

```diff
@@ -334,6 +390,76 @@ def test_git_py3_fresh_nodeps(self):
         self.assertEqual(json_response['right_color'], '#44CC44')
         self.assertLinkUrl(package, json_response['whole_link'])
 
+    def test_pypi_py2py3_fresh_nodeps(self):
```

Comment:
You need to pick names to indicate what you are testing. Maybe:
`test_pypi_py2py3_fresh_nodeps_failure_in_unsupported_version_combination`
   "Tests that failures in pair compatibility results for unsupported Python versions are ignored." 

But too long...

   """Tests that irrelevant pair compatibility results are ignored.

   Pair compatibility results for unsupported Python versions should be ignored.
   For example, the pair <apache-beam, google-api-core> for Python 3 should
   be ignored because apache-beam does not support Python 3. 
   """

Reply:
Actually, what is this testing? And this duplicates an existing name?

Reply:
This used to be `test_pypi_mix_fresh_nodeps` but you mentioned in an earlier comment that `mix` should be changed to `py2py3` but I didn't realize that would cause a naming conflict. I changed the name to something self evident.